.circleci/config.yml

```diff
@@ -19,7 +19,6 @@ jobs:
       # when movign back to yarn, also the v1-dependencies key should be set back to `yarn-lock`
       # - run: yarn install --pure-lockfile
       - run: npm install
-      - run: npm install web3@1.0.0-beta.29 --save
```

Comment:
By adding this dependency explicitly to `package.json/package-lock.json` we don't need the extra step after `npm install`